### PR TITLE
[stable/grafana] Correctly set options.path in dashboardProviders

### PR DESCRIPTION
**Pull request by Segflow**
`path` must be `/var/lib/grafana/dashboards/<provider_name>` while the example is `/var/lib/grafana/<provider_name>/dashboards/`

### CODE REVIEW

**Review by costimuraru — APPROVED** [review]

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.12.0
+version: 1.12.1
```
Comment (davidkarlsen) [inline stable/grafana/Chart.yaml]: Version needs bumping again after previous merges.

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Pull request merged**